packages/frontend/src/features/scheduler/components/SchedulerModalContent.tsx

```diff
@@ -217,7 +220,7 @@ const UpdateStateContent: FC<{
                 confirmText="Save"
                 onBack={onBack}
                 onSendNow={handleSendNow}
-                loading={mutation.isLoading || scheduler.isLoading}
+                loading={mutation.isLoading || scheduler.isInitialLoading}
```

Comment:
check if `mutation.isLoading` is correct

Reply:
@joaoviana  mutation states are the same, they don't have `isInitialloading` so no need to update it

Reply:
amazing!